tasks/ca-certificates.yml

```diff
@@ -29,3 +29,18 @@
     keystore_pass: "{{ ca_certificates.password }}"
     state: present
   with_items: "{{ ca_certificates.certificates }}"
+
+# Workaround for https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=894979
+# This only happens on Stretch with Java 11 via stretch-backports.
+# This bug is fixed in ca-certificates-java in Buster.
+- name: Check format of keystore (stretch-backports)
+  command: "file /etc/ssl/certs/java/cacerts"
```

Comment:
Note that `stat` module provide a `get_mime` parameter (and `stat` module handles check mode).

Reply:
I modified the code to use `stat`, I didn't know the module was able to return the mime type ! Thanks !